### ISSUE TITLE
Fabricators have "insufficient power" if left creating something when the round ends

### ISSUE THREAD

**Issue by Balls2zawall** [Regalis11/Barotrauma#3438]

**Comment by thegrb93** [comment]
I've seen this before. If you click the button anyway, it'll start working. The button text just hasn't updated or something/

**Comment by thegrb93** [comment]
Also this has been happening since version 0.9 so it's not new.

**Comment by mygamingaccount** [comment]
This is purely cosmetic, gameplay is not effected.

**Comment by mygamingaccount** [comment]
Duplicate issue of https://github.com/Regalis11/Barotrauma/issues/2214

**Comment by Regalis11** [comment]
Fixed in https://github.com/Regalis11/Barotrauma-development/commit/45c14ed465956a4b1fffd97b879f0e1640f796f0

**Comment by Rokvach** [comment]
This seems to be still broken,

Tested by launching a multiplayer campaign round with Typhon 2 and started crafting a Body Armor, when the crafting started I pressed the button to leave the outpost. Now once the mission loaded the Fabricator isn't crafting anything anymore and pressing the "Create" button doesn't do anything. If I try to remove the items from the input slots they will just move back to the slots from my inventory. 

The crafted item did show up as being done once the crafting time elapsed. I am now able to craft new items with the Fabricator. 
So I think the Fabricators state is desynced, meaning it should still be crafting but I see it as doing nothing. 

@Regalis11
